ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -30,7 +31,7 @@ import           Ouroboros.Consensus.Util (repeatedly)
 openMempool :: (MonadSTM m, ApplyTx blk)
             => ChainDB m blk hdr
             -> LedgerConfig blk
-            -> m (Mempool m blk)
+            -> m (Mempool m blk TicketNo)
```

Comment:
:+1: yeah, so our impl uses a concrete type for the idx, while the API remains general